src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/RegionalProcessingCenter.java

```diff
@@ -32,25 +33,28 @@ public RegionalProcessingCenter(@JsonProperty("faxNumber") String faxNumber,
         this.postcode = postcode;
         this.city = city;
         this.email = email;
+        this.hearingRoute = hearingRoute;
     }
 
-    private String faxNumber;
```

Comment:
Why'd you remove all the access modifiers?

Reply:
We are using the lombok @Value which set the access profile to private anyway so seemed a bit redundant to have it twice